### DISCUSSION TITLE
why create a global Ort::Env object will raise an error ？

### DISCUSSION THREAD

**Discussion by xlb8** [microsoft/onnxruntime#7123] [Other Q&A]
**Describe the bug**
maybe not a error？

Ort::Env env{ ORT_LOGGING_LEVEL_WARNING, "test" }; //global block
raise--->Ort::GetApi(...) return nullptr 
works well  in local block.

**Urgency**
none

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):win10
- ONNX Runtime installed from (source or binary):nuget package
- ONNX Runtime version:1.7.0
- Python version:
- Visual Studio version (if applicable):2019
- GCC/Compiler version (if compiling from source):ms
- CUDA/cuDNN version:
- GPU model and memory:

**To Reproduce**
- Describe steps/code to reproduce the behavior.
- Attach the ONNX model to the issue (where applicable) to expedite investigation.

**Expected behavior**

**Screenshots**

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

**Comment by snnn** [comment]
https://isocpp.org/wiki/faq/ctors#static-init-order
